File: Bridge/Resources/.generated/system/stringComparer.js
    Bridge.define("System.StringComparer", {
        inherits: [System.Collections.Generic.IComparer$1(System.String),System.Collections.Generic.IEqualityComparer$1(System.String)],
        statics: {
            props: {
                CurrentCulture: {
                    get: function () {
                        throw new System.NotImplementedException();
                    }
                },
                CurrentCultureIgnoreCase: {
                    get: function () {
                        throw new System.NotImplementedException();
                    }
                },
                InvariantCulture: {
                    get: function () {
                        throw new System.NotImplementedException();
                    }
                },
                InvariantCultureIgnoreCase: {
                    get: function () {
                        throw new System.NotImplementedException();
                    }
                },
                Ordinal: {
                    get: function () {
                        throw new System.NotImplementedException();
                    }
                },
                OrdinalIgnoreCase: {
                    get: function () {
                        throw new System.NotImplementedException();
                    }
                }
            },
            methods: {
                create: function (culture, ignoreCase) {
                    throw new System.NotImplementedException();
                }
            }
        },
        ctors: {
            ctor: function () {
                this.$initialize();
            }
        },
        methods: {
            compare$1: function (x, y) {
                throw new System.NotImplementedException();
            },
            equals: function (x, y) {
                throw new System.NotImplementedException();
            },
            getHashCode: function (obj) {
                throw new System.NotImplementedException();
            }
        }
    });

File: Bridge/System/StringComparer.cs
using Bridge;
using System.Collections.Generic;
using System.Globalization;

namespace System
{
    public abstract class StringComparer : IComparer<string>, IEqualityComparer<string>
    {
        protected StringComparer()
        {
        }

        public static StringComparer CurrentCulture
        {
            get
            {
                throw new NotImplementedException();
            }
        }

        public static StringComparer CurrentCultureIgnoreCase
        {
            get
            {
                throw new NotImplementedException();
            }
        }

        public static StringComparer InvariantCulture
        {
            get
            {
                throw new NotImplementedException();
            }
        }

        public static StringComparer InvariantCultureIgnoreCase
        {
            get
            {
                throw new NotImplementedException();
            }
        }

        public static StringComparer Ordinal
        {
            get
            {
                throw new NotImplementedException();
            }
        }

        public static StringComparer OrdinalIgnoreCase
        {
            get
            {
                throw new NotImplementedException();
            }
        }

        public static StringComparer Create(CultureInfo culture, bool ignoreCase)
        {
            throw new NotImplementedException();
        }

        public int Compare(object x, object y)
        {
            throw new NotImplementedException();
        }

        public new bool Equals(object x, object y)
        {
            throw new NotImplementedException();
        }

        public int GetHashCode(object obj)
        {
            throw new NotImplementedException();
        }

        public abstract int Compare(string x, string y);
        public abstract bool Equals(string x, string y);
        public abstract int GetHashCode(string obj);
    }
}